### ISSUE TITLE
Active facet header has insufficient contrast

### ISSUE THREAD

**Issue by seanaery** [projectblacklight/blacklight#3560]
An active facet ends up highlighted with a dark green `#198754` header. This provides inaccessible contrast to the facet title `#f8f9fa` (4.29) and the expand/collapse arrow `#052C65` (2.98). Contrast ratio must be at least 4.5:1 for both.

**Comment by seanaery** [comment]
<img width="597" alt="Image" src="https://github.com/user-attachments/assets/59140a22-ee3c-4529-9774-a1031f6519bf" />

**Comment by jcoyne** [comment]
@seanaery might this be something to move to bootstrap? I don't think we're setting the success color, just using the default.

**Comment by jcoyne** [comment]
Another possiblity here is to make that font-size > 18px which qualifies it as "large text" in WCAG and then a contrast ratio of 3:1 is specified. In the older version of blacklight, we make the cheveron the same color as the text, which ought to help:

<img width="328" alt="Image" src="https://github.com/user-attachments/assets/8757b67f-26ef-49a4-bf8e-804e463ef7e0" />

**Comment by seanaery** [comment]
@jcoyne Those are good ideas to consider, thanks. But after looking closer, it looks like this issue's on us. The default Bootstrap UI does provide sufficient contrast when an accordion section is active ([example](https://getbootstrap.com/docs/5.3/components/accordion/#example)). The code that's applying the success color here is in Blacklight in [_facets.scss](https://github.com/projectblacklight/blacklight/blob/main/app/assets/stylesheets/blacklight/_facets.scss).

I think it's OK for us to use success, we just also need to refine the label and chevron colors. I hesitate to increase the `font-size` there as a solution for a couple reasons. The [WCAG 3:1 rule](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) is for `18pt` and up, which is bigger than `18px` -- about `24px` -- significantly bigger than the default body font size. I think we also want to accommodate flexibility for implementers who might have some long facet field names and/or layouts where the facet sidebar is narrow.
